### DISCUSSION TITLE
packages for picoreplayer?

### DISCUSSION THREAD

**Discussion by dflavio** [mrene/minidsp-rs#690] [Ideas]
is it possible to create packages for picoreplayer (tinycorelinux)?
Why is it one of the best audio players for rpi and work vell with minidsp DDRC24

**Comment by tiprya** [comment]
I think an official tinycore extension would be a great idea!

I don't know enough to be able to build that... but I did hack together a local extension that runs on my flex HT.

Here are the rough steps if you know a little command line:

- enable ssh on the picoreplayer 
- load the packages you need: `tce-load -i squashfs-tools findutils libusb` (I can't remember if these are on picoreplayer by default... if not add the `w` flag to download them)
- `mkdir /tmp/minidsp` make a directory for the files
- `cd /tmp/minidsp` move into that directory
- `wget` the .deb package built for your architecture from `https://github.com/mrene/minidsp-rs/releases` (e.g. my 64bit pi uses the arm64.deb)
- I copied the script from here to convert a deb to a tcz: `https://forum.tinycorelinux.net/index.php/topic,2325.msg172668.html#msg172668` (reply 40) and saved it as deb2tcz
- `chmod a+x deb2tcz` make the script executable
- `./deb2tcz [.deb filename] minidsp.tcz` make the tinycore extension
- `mv minidsp.tcz /mnt/[bootdrive]/tce/optional/` move to where extensions are saved
- `tce-load -i minidsp` this should load the extension 
- test by running `minidsp`

If you want to load this on boot: `echo 'minidsp.tcz' >> /etc/sysconfig/tcedir/onboot.lst`

I hope this is helpful... this was the result of me fiddling around for a couple of hours this morning...

**Reply by dflavio** [reply]
Very helpful!!!work well
thank you very much!!!
now i'm able to control my DDRC-24 connect to usb to raspi4 with picoreplayer from ssh
Unfortunately don't work control from Minidsp Device Console or mobile app even if I changed the configuration parameters on "config.toml" (very hard to do on Picoreplayer /Tinycore because read only on "etc/minidsp)
